src/Hintman.hs

```diff
@@ -2,5 +2,14 @@ module Hintman
        ( someFunc
        ) where
 
+import Hintman.Cli (Context(..), cliContext)
+
 someFunc :: IO ()
-someFunc = putStrLn ("someFunc" :: String)
+someFunc = cliContext >>= runOn
```

Comment:
I feel like `Hintman` module should just reexport single `runHintman` function that parse CLI and runs server. Could you do this in this PR as well?